homework_2/program.js

```diff
@@ -0,0 +1,22 @@
+var ArgStart;
```

Comment:
You can name your variables whatever you want, but the typical format is to make them camel-cased eg: argStart, argSum, etc.